internal/executor/processdumper/processdumper_test.go

```diff
@@ -0,0 +1,7 @@
+package processdumper
+
+import "testing"
+
+func TestDump(t *testing.T) {
+	Dump()
```

Comment:
Is there a need so some assertions?

Reply:
This is just a the smoke-test to double-test the underlying libraries on different OS'es.

Reply:
I've thought of checking the dump line for something like the init daemon with PID 1, but this init daemon's comm/exe will be different on each platform, unfortunately.